### ISSUE TITLE
Wrong validator voluntary exit success URL when network is goerli

### ISSUE THREAD

**Issue by geldmon** [prysmaticlabs/prysm#13211]
### Describe the bug

Upon successfully exiting a goerli validator, the tracking URL is wrong:
```INFO accounts: Voluntary exit was successful for the accounts listed. URLs where you can track each validator's exit:
https://beaconcha.in/validator/<validator_pubkey>
```
Should be instead: 
```
https://prater.beaconcha.in/validator/<validator_pubkey>
```

### Has this worked before in a previous version?

```text
N/A
```

### 🔬 Minimal Reproduction

Run and complete the steps from:
```sh
./prysm.sh validator accounts voluntary-exit  --public-keys <validator_pubkey> 
```

### Platform(s)

Linux (x86)

### What version of Prysm are you running? (Which release)

v4.1.1

**Comment by v4lproik** [comment]
Hi! I would love to take this up :)

**Comment by rkapka** [comment]
Hi @v4lproik , I assigned it to you. Thanks a lot for volunteering!

**Comment by v4lproik** [comment]
@geldmon, could you please share your output? It appears that the command you provided in the description produces the intended output as the 'goerli' flag is not passed to `prysm.sh`.

It seems that the environment flag is being correctly taken into account when passed as parameter.

While investigating, I realised that there's no test coverage for the piece of code formatting the beaconchaURL. I added some tests for that and also included support for Sepolia and Holesky.

**Comment by geldmon** [comment]
I didn't pass the flag, that's fine then.
Lastly, I wonder if this shouldn't be attainable already during the exit process (i.e when calling beacon chain)

**Comment by prestonvanloon** [comment]
Fixed by #13308
